### PR TITLE
March update

### PR DESCRIPTION
**Pull request by joelhulen**
Addresses #13 

Please review.

### CODE REVIEW

**Comment by joelhulen** [comment]
Latest commit closes #11, #14, and #15.

**Comment by kartben** [comment]
There is a lack of consistency in the dimensions of the various screen captures (see e.g Task 3 of the HOL, but this is the case in several other areas), which makes it hard for the reader to see the "flow" between each step. It would be great if this could be fixed.

**Comment by joelhulen** [comment]
> There is a lack of consistency in the dimensions of the various screen captures (see e.g Task 3 of the HOL, but this is the case in several other areas), which makes it hard for the reader to see the "flow" between each step. It would be great if this could be fixed.

@kartben Could you please point to which Exercise the Task 3 you are referring to falls under? Also, do you prefer smaller or larger images? Or by dimensions, are you talking about wider vs. taller ratios? In that case, there's little that can be done depending on the composition of the contents that are being captured. Just trying to understand what you are referring to.

**Comment by Teodelas** [comment]
In WDS trainer guide - Internet of Things.md, this is not correct:
 "IoT Hub Message Routing might also be used, but this has the limitation that messages follow a single path, so with the current implementation, it would not be possible to send all messages to the cold path, while simultaneously sending some of the same messages into the hot path."

IoT Message Routing supports multiple routes.  This, along with consumer groups, allows for data to go to multiple paths / applications.

**Comment by Teodelas** [comment]
In WDS trainer guide - Internet of Things.md, this is not correct:
 "IoT Hub Message Routing might also be used, but this has the limitation that messages follow a single path, so with the current implementation, it would not be possible to send all messages to the cold path, while simultaneously sending some of the same messages into the hot path."

IoT Message Routing supports multiple routes.  This, along with consumer groups, allows for data to go to multiple paths / applications.

File: Hands-on lab/Before the HOL - Internet of Things.md [hunk 1]
```diff
@@ -78,6 +80,8 @@ In this task, you will create an Azure resource group for the resources used thr
 
 In this task, you will provision a virtual machine running Visual Studio Community 2017 that you will use as your development machine for this hands-on lab.
 
+> Note: Your Azure subscription must have MSDN offers associated with it to provision a new virtual machine with Visual Studio pre-loaded. If your subscription does not meet this requirement, you will need to either create a new VM with the same settings below, but without Visual Studio pre-installed, then install Visual Studio Community 2017 15.6 or later, or install it on your own machine.
```
Comment (mpeder) [inline Hands-on lab/Before the HOL - Internet of Things.md]: Shouldn't we mention that they can also just use their own laptop if they have VS 2017 or newer installed?

**Comment by DawnmarieDesJardins** [comment]
@joelhulen 
Hi Joel!  We have lead approval to finalize this update.  Please reassign to me when it's ready for QC/merge.
Thanks, Dawnmarie

File: Hands-on lab/Before the HOL - Internet of Things.md (hunk 1 above)
Comment (mrfalafel) [reply Hands-on lab/Before the HOL - Internet of Things.md]: You can only have ONE query per SA job. If you can share a way to have TWO queries on the same SA Job, I am happy to update the MCW.

**Pull request merged**